metrictank.go

```diff
@@ -417,6 +392,10 @@ func main() {
 	// When the timer becomes 0 it means the in-memory buffer has been able to fully populate so that if you stop a primary
 	// and it was able to save its complete chunks, this node will be able to take over without dataloss.
 	// You can upgrade a candidate to primary while the timer is not 0 yet, it just means it may have missing data in the chunks that it will save.
+	highestChunkSpan := chunkSpan
```

Comment:
^^ That's between the comment on line `391 - 394` and the code the comment is talking about